website/templates/home.mako

```diff
@@ -0,0 +1,5 @@
+<%inherit file="base.mako"/>
+<%def name="title()">Home</%def>
+<%def name="content()">
+    <h1>Placeholder</h1>
+</%def>
```

Comment:
This placeholder section will change obviously; further work will be done here but for this PR this is accurate.